torch/_inductor/output_code.py

```diff
@@ -307,14 +427,44 @@ def post_compile(
         cudagraphs: BoxedBool,
         gm: GraphModule,
     ) -> None:
-        # TODO: maybe move this here?  Not sure.
-        from torch._inductor.codecache import FxGraphCache
+        """
+        Run a set of post processing steps after loading from the cache. These involve:
+         - Setting the tracing context output strides
+         - Running cudagraphs if enabled
+         - Realigning inputs
 
-        FxGraphCache.post_compile(self, example_inputs, cudagraphs, gm)
+        This runs whether or not we have a cache hit, and always runs directly after we get a CompiledFxGraph.
```

Comment:
I see, this refactor is nice because OutputGraph is at a layer higher than a cache, so it can do the post compilation unconditionally. Without OutputCode, this refactor would be hard to write because there would be two different ways you could obtain all the needed inputs: through compile or through cache hit.


That said, maybe I'm missing context from the last stack but I don't remember how the deserialize step converts the stuff saved into the cache to an outputcode. I assume we still need something like that, to massage the cached result into the object we return from inductor.